### DISCUSSION TITLE
Mention what version of node it works on and doesn't in the README?

### DISCUSSION THREAD

**Discussion by max-sym** [geckosio/geckos.io#259] [Ideas]
I was having problems connecting to the server using node@19 - it would listen, but not connect. Even the simple chat example would hang, until I downgraded to node@16 and it worked.
Would that be good to mention that information in README?

**Comment by yandeu** [comment]
I have just added Node v17 and v19 to the test.
https://github.com/geckosio/geckos.io/actions/runs/5067885274

Based on the Supported Platforms Table (https://github.com/murat-dogan/node-datachannel#supported-platforms), both should work.

Anyways, I always recommend using LTS versions. 16 and 18; and soon 20.

**Reply by max-sym** [reply]
Hmm alright, let me check. Maybe I did something wrong.

**Reply by max-sym** [reply]
Seems to be working with 18 and 20 too, so node wasn't the issue...
I think I simply forgot to restart the terminal after specifying openssl global env variables by changing

~/.bash_profile, ~/.bashrc, and ~/.zshrc for Zsh

```
export OPENSSL_ROOT_DIR=/usr/local/opt/openssl@1.1
export OPENSSL_CRYPTO_LIBRARY=/usr/local/opt/openssl@1.1/lib
export OPENSSL_INCLUDE_DIR=/usr/local/opt/openssl@1.1/include
```

I only realized that might have been the issue when downgrading to some lower version of geckos (like 1.7.0?) when it threw an error saying that it can't find those vars.
Maybe that also should be mentioned in the README?

**Reply by max-sym** [reply]
Just opened a PR https://github.com/geckosio/geckos.io/pull/260 to add this to README. Let me know if that works :)
